.github/workflows/ci.yaml

```diff
@@ -8,22 +8,10 @@ jobs:
     timeout-minutes: 45
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2.4.0
-        with:
-          node-version: 14
-          cache: 'yarn'
-      - run: yarn install --ci
+      - uses: ./.github/workflows/setup-env
```

Comment:
Nice, I never realised you could do that :)

Reply:
Neither did I

Reply:
It's a lot cleaner like this 👍